webpack/assets/javascripts/react_app/API.jsx

```diff
@@ -0,0 +1,11 @@
+import ServerActions from "./actions/ServerActions"
```

Comment:
At this point yes. If the file becomes too large it should be broken up. This is what I have seen in the flux implementations that I studied.